### DISCUSSION TITLE
Background too dark can this be adjusted?

### DISCUSSION THREAD

**Discussion by DROIDshield** [jurialmunkey/skin.arctic.horizon.2#822] [Discussion]
Just wondering there is away of adjuststing The AH2 screens or contrast im see the background as dark.

I think the background is way too dark what do you think?
![20230214_141138](https://user-images.githubusercontent.com/97023369/218909823-08d210d1-fbca-442b-87cf-58dab03bc68d.jpg)

**Comment by jurialmunkey** [comment]
It looks like you're using the "High Contrast" colour scheme. Use the normal themes instead and it won't be as dark

**Reply by DROIDshield** [reply]
Hey thanks man, I didn't realise that setting controlled the the screen background. I thought it only controlled the skin setting from light to dark. Now using "darker" as I still want my skin colour to be dark but the background fanart is now much clearer. 
![20230215_140353](https://user-images.githubusercontent.com/97023369/219048951-218c8ce1-2678-4202-bfa4-b2bbc3c2f4b7.jpg)
